components/Navigation/NavigationContainer/index.module.scss

```
@@ -0,0 +1,105 @@
+.navigation,
+%navigation {
+  background-color: var(--color-fill-side-nav);
+  box-sizing: border-box;
+  clip-path: polygon(0 0, 100% 0, 100% 100%, 0 100%);
+  height: calc(100vh - var(--nav-height));
+  max-width: 36rem;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  padding: 0 var(--space-20);
+  position: sticky;
+  top: var(--nav-height);
+  width: 100%;
+  z-index: 998;
+
+  @media (max-width: 900px) {
+    height: 42px;
+    max-width: unset;
+    overflow-y: hidden;
+    padding-bottom: var(--space-48);
+    position: relative;
+    top: 0;
+    transition: all 0.5s cubic-bezier(0.23, 1, 0.32, 1);
+
+    > div {
+      display: none;
+    }
+  }
+
+  &Open,
+  &%Open {
+    background: var(--color-text-primary);
+    background-clip: content-box;
+    border: none;
+    color: var(--color-text-primary);
+    display: none;
+    font-weight: var(--font-weight-light);
+    height: calc(24px + 3px);
+    line-height: 0;
+    margin: var(--space-08) 0 0 calc(var(--space-08) * -1);
+    overflow: visible;
+    padding: 12px 0;
+    position: sticky;
+    text-indent: 3.2rem;
+    transition: color 0.28s, background-color 0.28s;
+    user-select: none;
+    width: 20px;
+
+    @media (max-width: 900px) {
+      display: unset;
+    }
+
+    &::before {
+      background-color: var(--color-text-primary);
+      content: '';
+      height: 3px;
+      left: 0;
+      position: absolute;
+      top: calc(50% - 1.5px);
+      transform: translateY(-250%);
+      transition: transform 0.28s;
+      width: 100%;
+    }
+
+    &::after {
+      background-color: var(--color-text-primary);
+      content: '';
+      height: 3px;
+      left: 0;
+      position: absolute;
+      top: calc(50% - 1.5px);
+      transform: translateY(250%);
+      transition: transform 0.28s;
+      width: 100%;
+    }
+  }
+
+  &Fixed,
+  &%Fixed {
+    @media (max-width: 900px) {
+      height: 100vh;
+      overflow-y: scroll;
+      position: sticky;
+      top: 0;
+      z-index: 999;
+
+      > div {
+        display: block;
+      }
+    }
+  }
+
+  &Fixed &open {
```

```suggestion
  &Fixed &Open {
```
If we're going with this approach, we need a capital letter here too

---

I feel that we shouldn't rename the classnames. I'm pretty sure we can update the Sass configuration to do what we want.